icubam/db/synchronizer.py

```diff
@@ -192,7 +192,10 @@ def sync_bedcounts_from_csv(self, csv_contents: TextIO, force_update=False):
     col_diff = set(BC_COLUMNS) - set(bedcounts_df.columns)
     if len(col_diff) > 0:
       raise ValueError(f"Missing columns in input data: {col_diff}.")
-    self.sync_users(bedcounts_df, force_update)
+    bedcounts_df['create_date'] = pd.to_datetime(
+      bedcounts_df['create_date']
+    ).dt.tz_localize("Europe/Paris").dt.tz_convert(tz.tzutc())
```

Comment:
I had tried to set the TZ in the import script to not hard-code it her.  Could also pass in the tz string, but probably best to not fix it to Europe/Paris.

Reply:
Good point. What I don't understand in the previous code is that the datetime does not seem to be parsed here. Doesn't it always fail on any result from `read_csv`?

So maybe I should keep the parsing but not the time conversion. Adjustment of TZ should be done by a preprocessor.

Reply:
Should be better now. I struggled a bit because pandas and dateutils don't use the same object to represent UTC, it seems...